Giovaniavs/MyProject-Client
# Feature/footer

## ***What I Did:***
- New footer added and its stylizations.

## ***How to Test:***
1. Run this current branch and go the bottom of the page and see if there's a footer containing the text and icons
